spec/support/features/page_object/topics.rb

```diff
@@ -149,7 +149,7 @@ def someone_updates_topic
 
     def search_for(title)
       fill_in 'Search', with: title
-      find('.search-bar input[type="submit"]').click
+      find('.user-navigation--search input[type="submit"]').click
```

Comment:
Running and adjusting the failing tests.

Marry me.

You're a designer from the heavens.

Thank you thoughtbot.